rllib/policy/torch_policy.py

```diff
@@ -259,6 +264,8 @@ def _compute_action_helper(self, input_dict, state_batches, seq_lens,
         extra_fetches = self.extra_action_out(input_dict, state_batches,
                                               self.model, action_dist)
 
+        self._lock.release()
```

Comment:
Hmm can we do the release in a finally block to avoid hangs? This might cause a lot of indentation changes, so one strategy is to use an method annotation (e.g.,

```
@with_lock
def _compute_action_helper:...

```